### DISCUSSION TITLE
Blockbreak not registering

### DISCUSSION THREAD

**Discussion by sqcred** [Minestom/Minestom#672] [Help] [answered]
My BlockBreakListener is not being called when my pickaxe has efficency 10. Any idea why?

**Comment by TheMode** [comment, marked as answer]
Relate to https://github.com/Minestom/Minestom/issues/289
We unfortunately do not properly handle insta break yet.

**Reply by sqcred** [reply]
ahhh i see well thanks for the fast answer
